mitodl/hugo-course-publisher
# remove 3rd tier nav items not flagged

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/IHLhRE3H/159-dont-include-third-level-pages-in-the-left-nav-unless-listinleftnav-is-set-to-true

#### What's this PR do?
This PR updates the sample markdown to remove 3rd tier nav items not explicitly marked to appear with `list_in_left_nav` set to true.

#### How should this be manually tested?
Run the site or visit the deploy preview.  Find a course in production that has 3rd tier nav items where there shouldn't be, i.e. https://ocw-next.netlify.app/courses/res-15-003-shaping-the-future-of-work-15-662x-spring-2016/sections/introduction-challenges-and-opportunities/ and verify that they have been removed.

#### Any background context you want to provide?
Other relevant PR's:
https://github.com/mitodl/ocw-data-parser/pull/41
https://github.com/mitodl/ocw-to-hugo/pull/50

#### Screenshots (if appropriate)
Before:
![Screen Shot 2020-07-13 at 5 17 13 PM](https://user-images.githubusercontent.com/12089658/87354585-c99d8d00-c52c-11ea-9682-5dfc5275b76c.png)
![Screen Shot 2020-07-13 at 5 21 59 PM](https://user-images.githubusercontent.com/12089658/87354913-695b1b00-c52d-11ea-9896-1d4e40966345.png)

After:
![Screen Shot 2020-07-13 at 5 16 48 PM](https://user-images.githubusercontent.com/12089658/87354600-d0c49b00-c52c-11ea-87d0-66da7353a8b4.png)
![Screen Shot 2020-07-13 at 5 20 49 PM](https://user-images.githubusercontent.com/12089658/87354922-6f50fc00-c52d-11ea-9c09-92de2e6b5dd3.png)
